x-pack/plugin/core/src/main/resources/org/elasticsearch/xpack/entsearch/analytics/behavioral_analytics-events-default_policy.json

```diff
@@ -4,7 +4,7 @@
       "actions": {
         "rollover": {
           "max_age": "30d",
-          "max_size": "3GB"
+          "max_primary_shard_size": "50gb"
```

Comment:
ℹ️ Using the same settings than those used in the default [log policy](https://github.com/elastic/elasticsearch/blob/8d0fe6f498ddf5072959e1ace1d49eefd540ae34/x-pack/plugin/core/src/main/resources/logs-policy.json).